### ISSUE TITLE
Bazel fails to build due to missing symbol com.google.protobuf.ProtocolStringList

### ISSUE THREAD

**Issue by Ralith** [NixOS/nixpkgs#8194]
```
% nix-env -iA nixpkgs.bazel
installing ‘bazel-20150326.981b7bc1’
these derivations will be built:
  /nix/store/fijfxkrbc29fbqz2s017jrdcfx2pwxmw-bazel-20150326.981b7bc1.drv
building path(s) ‘/nix/store/16b80ympqg53ck9qv55h8yyrbhnrqzc2-bazel-20150326.981b7bc1’
unpacking sources
unpacking source archive /nix/store/fwg97l6ljarp0zb2sqgb0d44bylyvz4g-bazel-981b7bc1-src
source root is bazel-981b7bc1-src
patching sources
configuring
no configure script, doing nothing
building
no Makefile, doing nothing
installing
Compiling Java stubs for protocol buffers...
Compiling Bazel Java code...
src/main/java/com/google/devtools/build/lib/server/signal/InterruptSignalHandler.java:19: warning: Signal is internal proprietary API and may be removed in a future release
import sun.misc.Signal;
               ^
src/main/java/com/google/devtools/build/lib/server/signal/InterruptSignalHandler.java:20: warning: SignalHandler is internal proprietary API and may be removed in a future release
import sun.misc.SignalHandler;
               ^
output/src/com/google/devtools/build/lib/query2/proto/proto2api/Build.java:173: error: cannot find symbol
    public com.google.protobuf.ProtocolStringList
                              ^
  symbol:   class ProtocolStringList
  location: package com.google.protobuf
output/src/com/google/devtools/build/lib/query2/proto/proto2api/Build.java:202: error: cannot find symbol
    public com.google.protobuf.ProtocolStringList
                              ^
  symbol:   class ProtocolStringList
  location: package com.google.protobuf
output/src/com/google/devtools/build/lib/query2/proto/proto2api/Build.java:18: error: cannot find symbol
    com.google.protobuf.ProtocolStringList
                       ^
  symbol:   class ProtocolStringList
  location: package com.google.protobuf
...
```

CC @rbvermaa

**Comment by IreneKnapp** [comment]
The soluton here is to build against protobuf2_5 instead of just protobuf, which is the latest.

**Comment by pSub** [comment]
Fixed in 76dd430cd87c1fe9d859843c5d228abbdc7923bb.
